store/tikv/region_cache.go

```diff
@@ -416,6 +416,8 @@ func (c *RegionCache) OnRequestFail(ctx *RPCContext) {
 	delete(c.storeMu.stores, storeID)
 	c.storeMu.Unlock()
 
+	log.Warnf("drop regions from cache due to request fail, storeID: %v", storeID)
```

Comment:
Is it better to log this as info? This case is normal on bad network, leader change, region split, and so on....